config/services.php

```diff
@@ -59,6 +60,12 @@
         ])
     ;
 
+    $services->set(FileItemFactory::class)
+        ->args([
+            service('mime_types'),
```

Comment:
I think this needs a

```suggestion
            service('mime_types')->nullOnInvalid(),
```

as the `mime_types` service is defined by the `symfony/framework-bundle` and thus might not exist. Symfony itself also uses `->args([service('mime_types')->nullOnInvalid()])` in its various components.